static/usage/v7/select/customization/icon-flip-behavior/react/main_tsx.md

```diff
@@ -0,0 +1,40 @@
+```tsx
+import React from 'react';
+import { IonItem, IonList, IonSelect, IonSelectOption } from '@ionic/react';
+
+import './main.css';
+
+function Example() {
+  return (
+    <IonList>
+      <IonItem>
+        <IonSelect
+          className="always-flip"
+          toggleIcon="caret-down-sharp"
+          interface="popover"
+          label="Icon flips on both modes"
+          placeholder="Select fruit"
+        >
+          <IonSelectOption value="apples">Apples</IonSelectOption>
+          <IonSelectOption value="oranges">Oranges</IonSelectOption>
+          <IonSelectOption value="bananas">Bananas</IonSelectOption>
+        </IonSelect>
+      </IonItem>
+      <IonItem>
+        <IonSelect
+          className="never-flip"
+          toggleIcon="caret-down-sharp"
```

Comment:
Ionic React needs the icons imported from `ionicons/icons` and then passed to the props

```typescript
import { caretDownSharp } from 'ionicons/icons';

...

toggleIcon={caretDownSharp}
```